### DISCUSSION TITLE
BA5FBDDC-FEEA-F437-FC8E-1DF31AD6E4FB

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#50] [General]
# BA5FBDDC-FEEA-F437-FC8E-1DF31AD6E4FB

Tiangong is a modular space station that has been constructed and completed by China from 2021-2022. It has an orbital mass of 60 tons, roughly about 1/5 the mass of the ISS and have the same size as Mir. As of December 2022, about four long-term Shenzhou expeditions have stayed aboard the station, each staying about six months aboard starting with the second expedition

https://celestia.mobi/addon?item=BA5FBDDC-FEEA-F437-FC8E-1DF31AD6E4FB&lang=zh_CN

**Comment by 13940951572** [comment]
现在空间站位置和软件上的空间站位置不符！请求改进！

**Reply by DaveBowman2001** [reply]
As I explained with a similar case to my [International Space Station](https://celestia.mobi/resources/item?item=DB2CB6F5-247F-761C-4001-072A6C6BE5FD) addon, this is simply not possible to do due to limitations of Celestia's own programming
